src/stylesheets/reset.less

```diff
@@ -1,28 +0,0 @@
-/* Eric Meyer's Reset CSS v2.0 - http://cssreset.com */
```

Comment:
Why do we need this at all if we're in an iframe? Isn't style already reset? Or is this meant to account for browser inconsistencies?